contact-page.css

```diff
@@ -64,18 +57,15 @@ h2 {
 }
 
 .form-box {
-    top: 10px;
-    margin: 50px 200px;
-    justify-items: center;
+    padding: 1rem 2rem;
 }
 
-.form-box #error-msg {
+.form-box .error-msg {
     margin: 10px;
-    padding: 0;
+    padding: 10px;
     background: #fe8bef;
-    text-align: center;
-    font-size: 14px;
-    transition: all 0.5s ease;
+    font-size: 1rem;
+    transition-property: all 0.5s ease;

```

Comment:
It won't work like this, you don't even change anything so you don't need to use transition.